browser/Makefile.am

```diff
@@ -411,6 +411,7 @@ COOL_JS_LST =\
 	src/control/Control.Permission.js \
 	src/control/Control.Selection.js \
 	src/control/Control.ServerAuditDialog.ts \
+	src/control/Control.Shortcuts.js \
```

Comment:
can we convert it to .ts ? we try to migrate all the code to TypeScript

Reply:
we can put it in `browser/src/control/jsdialog/Util.Shortcuts.ts`
as it will be used only in our UI library possibly